src/Symfony/Component/Console/Input/InputOption.php

```diff
@@ -42,7 +42,7 @@ class InputOption
     public const VALUE_IS_ARRAY = 8;
 
     /**
-     * The option accepts multiple values (e.g. --dir=/foo --dir=/bar).
+     * The option may have either positive or negative value.
```

Comment:
The other constants are documented with a small example. I think we should do the same here.